### PR TITLE
Update definition files

### PR DESCRIPTION
**Pull request by AroneyS**
In progress

### CODE REVIEW

**Comment by AroneyS** [comment]
Ready for merge.

File: resources/pathways_refined.tsv [hunk 1]
```diff
@@ -1,49 +1,57 @@
...
-nitrogen_redox	13	nitrite_oxidation	nxrA	K00370	
+nitrogen_redox	1+2,1+3+15	nitrate_reduction			60% of the KOs for all KOs in reaction
+nitrogen_redox	3+4,3+5+6	denitrification			70% of the reactions in a pathway need to have some presence
+nitrogen_redox	7	nitrogen_fixation			Y/N on the signature definition
+nitrogen_redox	8+9	anammox			
+nitrogen_redox	10+11	ammonia_oxidation			
+nitrogen_redox	13	nitrite_oxidation			
 nitrogen_redox	14	nitric_oxide_oxidation			
-nitrogen_redox	10+11+13	comammox	amoA+nxrA	K10944+K00370	
+nitrogen_redox	10+11+13	comammox	(pmoC,pmoA,pmoB,hao)+(nxrA,nxrB)	(K10946,K10944,K10945,K10535)+(K00370,K00371)
```
Comment (AroneyS) [inline resources/pathways_refined.tsv:9]: This signature definition has brackets. Will this still work?

Comment (rmFlynn) [reply resources/pathways_refined.tsv:9]: Yes it should, I can't recall if I tested it though.

**Comment by AroneyS** [comment]
I got an error: `ModuleNotFoundError: No module named 'openpyxl'`. So I added it to the environment.yaml.


I also get several thousand such warnings:

```
emerge_pipline/workflow/scripts/emerge_distill_tools/distill_genomes1_3.py:42: PerformanceWarning: DataFrame is highly fragmented.  This is usually the result of calling `frame.insert` many times, which has poor performance.  Consider joining all columns at once using pd.concat(axis=1) instead. To get a de-fragmented frame, use `newframe = frame.copy()`
  genome_summary_frame[genome] = counts
```

**Comment by AroneyS** [comment]
I now get the following error:
```
  File "emerge_pipline/workflow/scripts/paths_graph_parser.py", line 169, in make_path_graph
    raise ValueError("Index is not unique, make it so!")
ValueError: Index is not unique, make it so!
```

This index appears to be the gene ids. They seem to be unique in EMERGE_distillate_modules.tsv and in the distillate index prior to pooling. 

At this line:
https://github.com/WrightonLabCSU/emerge_pipline/blob/d36c5e9f510c1b724ece40f4eb2d5ea162753b04/workflow/scripts/paths_graph_parser.py#L428

**Comment by AroneyS** [comment]
Actually might be index for the series created by each thread. In which case it is the pathway-subpathway index. But these too appear to be unique.

At this line:
https://github.com/WrightonLabCSU/emerge_pipline/blob/d36c5e9f510c1b724ece40f4eb2d5ea162753b04/workflow/scripts/paths_graph_parser.py#L418

**Comment by rmFlynn** [comment]
Yeah I don't recall what this is I will tested first thing in the a.m. I'll remove that warning too probably, just needs an assignment statement somewhere I suspect.

**Comment by rmFlynn** [comment]
FYI the problem was that i used `-` as a delimiter for the reaction number, now it is `number:`

**Comment by AroneyS** [comment]
Thanks for the fix!

**Pull request merged**